src/main/java/com/leetcode/practice/easy/binary_search/BinarySearch.java

```diff
@@ -0,0 +1,38 @@
+package com.leetcode.practice.easy.binary_search;
+
+import java.util.Arrays;
+
+public class BinarySearch {
+
+    public boolean isTargetNumberPresent(int[] arr, int target) {
+        /**
+         * Search the target element in the given array and return true if present else return false
+         * @param arr array of integers
+         * @param target element to be searched
+         * @return boolean
+         * @TimeComplexity O(nlogn)
+         * @SpaceComplexity O(n)
+         * @TimeTaken 30minutes
+         *
+         * Example
+         */
+        Arrays.sort(arr);
```

Comment:
sorting this before checking the corner case will decrease the performance. This should be done after first if check